locations/spiders/denmans_gb.py

```diff
@@ -22,12 +22,8 @@ def parse(self, response):
                     ],
                 )
             )
-            store["address"]["country"] = store["address"]["country"]["isocode"]
+            store["ref"] = store.pop("name")
```

Comment:
That's a neat solution!

Reply:
It would have been better if I had changed the website at the same time 🤦

It works because DictParser will still get the name from the displayName field.